exercises/acronym/example.py

```diff
@@ -2,5 +2,5 @@
 
 
 def abbreviate(words):
-    regex = '[A-Z]+[a-z]*|[a-z]+'
+    regex = '[A-Z]+[\'a-z]*|[\'a-z]+'
```

Comment:
Again, switching to double quotes for this string literal removes the need for escaping apostrophes.